Packs/FraudWatch/Integrations/FraudWatch/FraudWatch.py

```diff
@@ -0,0 +1,832 @@
+import pytz
+import urllib3
+
+import demistomock as demisto  # noqa: F401
+from CommonServerPython import *  # noqa: F401
+from requests import Response
+
+# disable insecure warnings
+urllib3.disable_warnings()
+
+''' CONSTANTS '''
+MINIMUM_POSITIVE_VALUE = 1
+MAX_LIMIT_VALUE = 200
+DEFAULT_LIMIT_VALUE = 50
+BASE_URL = 'https://www.phishportal.com/v1/'
+
+FRAUD_WATCH_DATE_FORMAT = '%Y-%m-%d'
+
+INCIDENT_LIST_MARKDOWN_HEADERS = ['identifier', 'reference_id', 'url', 'status', 'type', 'brand', 'client',
+                                  'content_ip', 'host', 'host_country', 'host_timezone', 'created_by', 'discovered_by',
+                                  'current_duration', 'active_duration', 'date_opened', 'date_closed',
+                                  'additional_urls', 'link']
+
+UTC_TIMEZONE = pytz.timezone('utc')
+''' CLIENT CLASS '''
+
+
+class Client(BaseClient):
+    URL_ENCODED_HEADER = {'Content-Type': 'application/x-www-form-urlencoded'}
+    JSON_CONTENT_HEADER = {'Content-Type': 'application/json'}
+
+    def __init__(self, api_key: str, base_url: str, verify: bool, proxy: bool):
+        self.api_key = api_key
+        self.base_headers = {
+            'Authorization': f'Bearer {api_key}',
+            'Accept': 'application/json'
+        }
+        super().__init__(base_url=base_url, verify=verify, proxy=proxy, headers=self.base_headers)
+
+    def http_request(self, method: str, url_suffix: str, params: Optional[Dict] = None, data: Optional[Dict] = None,
+                     headers: Optional[Dict] = None, files: Optional[Dict] = None):
+        """
+        Wrapper for Base Client http request. Uses error handler to catch errors returned by FraudWatch and returning
+        an exception with more precise description for the exception
+        Args:
+            method (str): The HTTP method to perform ('POST', 'GET', 'PUT').
+            url_suffix (str): The url suffix of the API call.
+            params (Optional[Dict]): Additional query params to be sent.
+            data (Optional[Dict]): Additional data to be sent.
+            headers (Optional[Dict]): Additional headers to be sent.
+            files (Optional[Dict]): Additional files to be sent
+
+        Returns:
+            The HTTP response, or exception if exception occurred.
+        """
+        return self._http_request(
+            method=method,
+            url_suffix=url_suffix,
+            params=params,
+            data=data,
+            headers=headers,
+            files=files,
+            error_handler=fraud_watch_error_handler
+        )
+
+    def incidents_list(self, brand: Optional[str], status: Optional[str], page: Optional[int],
```

Comment:
A small comment that I missed - For each param if it is indeed optional by the API then let's add `None` as the default value (`brand: Optional[str] = None`), If it is required then remove the replace `Optional[str]` > `str`.
Let's make sure this is true **for all client functions**.